clcache.py

```diff
@@ -192,7 +192,7 @@ def clean(self, stats, maximumSize):
         for o in objects:
             try:
                 objectInfos.append((os.stat(o), o))
-            except WindowsError:
+            except OSError:
```

Comment:
Interesting, thanks for digging that up! I'm surprised Ubuntu didn't complain about the

```python
from ctypes import windll, wintypes
```

at the top of `clcache.py`!